snooty/parser.py

```diff
@@ -678,6 +680,11 @@ def handle_directive(
                     self.diagnostics.append(InvalidURL(line))
                 return doc
 
+            if api_version:
+                doc.options["source_type"] = "atlas"
+                doc.options["api_version"] = api_version
+                return doc
+
             if uses_realm:
```

Comment:
I have left :uses-realm: as a supported option for backwards compatibility. Going forward it might not be necessary at all, but just leaving the note here!